src/App.js

```diff
@@ -1,12 +1,23 @@
 import React, { Component } from 'react';
 import { Route, Link } from 'react-router-dom'
 import './css/index.css';
-import { headerNav } from './header.js'
+
+
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+//Import Material-UI
 
 // TODO : Import all the components
 import home from './components/home'
 
+import settings from './components/settings'
 
+import headerNav from './components/headerNav'
```

Comment:
I noticed your import names are not consistent some are camelcase, some pascal case, please decide on a standard